vcbuild.bat

```diff
@@ -240,6 +240,7 @@ if %target_arch%==x86 if %msvs_host_arch%==x86 set vcvarsall_arg=x86
 :vs-set-2019
 if defined target_env if "%target_env%" NEQ "vs2019" goto vs-set-2017
 echo Looking for Visual Studio 2019
+set "VCINSTALLDIR="
```

Comment:
```suggestion
@rem VS2017 is detected here when VCINSTALLDIR is set (as in VS Command Prompt)
set "VCINSTALLDIR="
```

Reply:
To be clear, this is just a suggestion. Feel free to take it, change it as you see fit or leave it.

Reply:
Added a (different) comment. PTAL.